composer.json

```diff
@@ -13,10 +13,12 @@
         "php": "^7.4 | ^8.0",
         "ext-bcmath": "*",
         "ext-json": "*",
+        "ext-libxml": "*",
         "ext-mbstring": "*",
         "ext-sodium": "*"
     },
     "require-dev": {
+        "ext-simplexml": "*",
```

Comment:
can you please add these extensions to all GitHub actions? the seem to be added by default, but I don't want the build to fail if configurations were to be changed in the future ( better safe than sorry 😛 )